independent-publisher-connectors/Fantasy Premier League/README.md

```diff
@@ -0,0 +1,44 @@
+# Fantasy Premier League Independant Connector
+The Fantasy Premier League independant connector allows you to gather information from your fantasy football(soccer) leagues, ratings, information on current fixtures, general information and more. This will allow you to gather information on your fantasy league and post information about your league, such as current standings, to your Microsoft Teams groupchat or channel.
+
```

Comment:
@Flow-Joe please add Publisher heading in readme.md file, it should look like `Publisher: Publisher's name`. You can refer to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md?plain=1) for more information about required headings.